### PR TITLE
Add Flood risk activity exemptions FO and BO pages

### PR DESCRIPTION
**Pull request by woodpigeon**
Add maintenance and unavailable pages for Flood risk activity exemptions FO and BO.

### CODE REVIEW

**Comment by Cruikshanks** [comment]
Looks good to me. 🚢 it!

File: back-office/frae-bo-maintenance.html [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+      <header>
+        <h1>Service unavailable</h1>
+      </header>
+      <div class="article-container">
+        <article role="article" class="group">
+          <div class="inner text">
+
+            <p>We're carrying out some work on the flood risk activity exemptions registration service.</p>
+
+            <p>Sorry about this. We’ll have it back up and running as soon as we can.</p>
```
Comment (reggieb) [inline back-office/frae-bo-maintenance.html:84]: Line 82 `We're`, but line 84 `We’ll` - that two different single quote/apostrophe characters `'` and `’`  - need to choose and use one

**Comment by Cruikshanks** [comment]
Bloody good spot @reggieb

File: back-office/frae-bo-maintenance.html (hunk 1 above)
Comment (reggieb) [inline back-office/frae-bo-maintenance.html:84]: I have a feeling that `'` is `&#39;` and part of the core HTML character set, but `’` is font specific. So personally, I'd use the former

**Pull request merged**